### PR TITLE
Article data byte fix

### PR DESCRIPTION
**Pull request by Taz-Z**

### CODE REVIEW

File: Project/src/main/java/app/Application.java [hunk 1]
```diff
@@ -37,6 +37,7 @@ public CommandLineRunner demo(ArticleRepository articleRepository, UserRepositor
             users.add(user1);
             users.add(user3);
             article.setUsers(users);
+            article.setData("TEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEST".getBytes());
```
Comment (br0wncrust4cean) [inline Project/src/main/java/app/Application.java]: needs another E, not enough

**Review by br0wncrust4cean — APPROVED** [review]

File: Project/src/main/java/app/Application.java (hunk 1 above)
Comment (DanGravel) [inline Project/src/main/java/app/Application.java]: Spelling

**Review by DanGravel — APPROVED** [review]

**Comment by br0wncrust4cean** [comment]
Oh I think you accidentally pushed a class file

**Review by br0wncrust4cean — CHANGES_REQUESTED** [review]
Get rid of the class file in here

**Review by br0wncrust4cean — APPROVED** [review]

**Pull request merged**